apps/arweave/src/ar_sqlite3.erl

```diff
@@ -197,15 +196,11 @@ handle_call({eval_legacy_arql, Query}, _, State) ->
 				"WHERE ", WhereClause,
 				" ORDER BY block.height DESC, tx.id DESC"
 			]),
-			case sqlite3:prepare(?DB_NAME, SQL) of
-				{ok, Stmt} ->
-					ok = sqlite3:bind(?DB_NAME, Stmt, Params),
-					R = collect_txids(Stmt),
-					ok = sqlite3:finalize(?DB_NAME, Stmt),
-					R;
-				{error, 1, "parser stack overflow"} ->
-					sqlite_parser_stack_overflow;
```

Comment:
Could you, please, also update `ar_http_iface_middleware.erl` - it has some special handling for `sqlite_parser_stack_overflow`.

Reply:
Added back 👍 ff3cf644